packages/cozy-stack-client/src/OAuthClientsCollection.js

```diff
@@ -0,0 +1,122 @@
+import get from 'lodash/get'
+
+import DocumentCollection from './DocumentCollection'
+import { uri } from './utils'
+import * as querystring from './querystring'
+import { dontThrowNotFoundError } from './Collection'
+import { FetchError } from './errors'
+
+export const OAUTH_CLIENTS_DOCTYPE = 'io.cozy.oauth.clients'
+
+const normalizeDoc = DocumentCollection.normalizeDoctypeJsonApi(
+  OAUTH_CLIENTS_DOCTYPE
+)
+const normalizeOAuthClient = client => ({
+  ...normalizeDoc(client, OAUTH_CLIENTS_DOCTYPE),
+  ...client.attributes
+})
+
+/**
+ * Implements `DocumentCollection` API to interact with the /settings/clients endpoint of the stack
+ */
+class OAuthClientsCollection extends DocumentCollection {
+  constructor(stackClient) {
+    super(OAUTH_CLIENTS_DOCTYPE, stackClient)
+  }
+
+  /**
+   * Fetches all OAuth clients
+   *
+   * @param  {object} options           Query options
+   * @param  {number} options.limit     For pagination, the number of results to return.
+   * @param  {object} options.bookmark  For cursor-based pagination, the index cursor.
```

Comment:
No documentation for `options.keys`. Is it on purpose?

Reply:
Hum no, I forgot to add it after I copy/pasted it.